kubespawner/spawner.py

```diff
@@ -1613,11 +1611,6 @@ def events(self):
 
             len_events = len(events)
             if next_event < len_events:
-                # only show messages for the 'current' pod
-                # pod_id may change if a previous pod is being stopped
-                # before starting a new one
-                # use the uid of the latest event to identify 'current'
-                pod_id = events[-1].involved_object.uid
```

Comment:
Hmmmmm...
- Every KubeSpawner get their own pod.
- my_spawner.events returns a list filtered, but this list is filtered to the pod, but perhaps the name of the pod rather than the UID of the pod?
- Two pods with the same name kmim the same namespace cannot happen.
- When does events disappear? I suspect it is after their involved object has been deleted, perhaps delayed or directly.
- Is this code legacy code, not working, meant to avoid getting events from past pods? Is this perhaps something to fix?

I know i dont want to remove this without make a remark about it in the code allowing someone to correct this later if needed with more information available to get started. So perhaps reference this comment?